docs/resources/repository_binding.md

```diff
@@ -1,67 +1,44 @@
 # cyral_repository_binding (Resource)
 
-Manages [repositories to sidecars binding](https://cyral.com/docs/sidecars/sidecar-assign-repo).
+Manages [cyral repository to sidecar bindings](https://cyral.com/docs/sidecars/sidecar-assign-repo).
 
 -> **NOTE** Import ID syntax is `{sidecar_id}/{repository_id}`.
```

Comment:
The import syntax changed to `{sidecar_id}/{binding_id}`, lets update the template with the new syntax.